<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ContractDialog</class>
 <widget class="QDialog" name="ContractDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>583</width>
    <height>471</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>30</number>
   </property>
   <property name="leftMargin">
    <number>20</number>
   </property>
   <property name="topMargin">
    <number>20</number>
   </property>
   <property name="rightMargin">
    <number>20</number>
   </property>
   <property name="bottomMargin">
    <number>25</number>
   </property>
   <item>
    <widget class="QLabel" name="label">
     <property name="frameShape">
      <enum>QFrame::Shape::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <property name="text">
      <string>Add/Edit Contracts</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::FieldGrowthPolicy::FieldsStayAtSizeHint</enum>
     </property>
     <property name="horizontalSpacing">
      <number>20</number>
     </property>
     <property name="verticalSpacing">
      <number>8</number>
     </property>
     <property name="leftMargin">
      <number>11</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="propertyIDLabel">
       <property name="text">
        <string>PropertyID:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="propertyidSpin">
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="clientIDLabel">
       <property name="text">
        <string>ClientID:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="clientidSpin">
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="agentIDLabel">
       <property name="text">
        <string>AgentID:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="agentidSpin">
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="priceLabel">
       <property name="text">
        <string>Price:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QDoubleSpinBox" name="priceSpin">
       <property name="maximum">
        <double>100000000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="startDateLabel">
       <property name="text">
        <string>Start Date</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="startDateEdit"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="endDateLabel">
       <property name="text">
        <string>End Date:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="endDateEdit"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="contractTypeLabel">
       <property name="text">
        <string>Contract Type:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="contractTypeEdit"/>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="isActiveLabel">
       <property name="text">
        <string>is Active?</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QCheckBox" name="activeCheck"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="saveContract">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Save</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>saveContract</sender>
   <signal>accepted()</signal>
   <receiver>ContractDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saveContract</sender>
   <signal>rejected()</signal>
   <receiver>ContractDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
